coreum/assetlist.json

```diff
@@ -61,6 +61,7 @@
       "name": "Ripple",
       "display": "xrp",
       "symbol": "XRP",
+      "coingecko_id": "ripple",
```

Comment:
Bridged tokens aren't allowed to reuse a coingecko_id. Only the original asset and additional-mintages can reuse a coingecko id.